crates/types/src/config/common.rs

```diff
@@ -56,6 +57,12 @@ pub struct CommonOptions {
 
     /// If true, then a new cluster is bootstrapped. This node *must* have an admin
     /// role and a new nodes configuration will be created that includes this node.
+    ///
+    /// bootstrap is allowed by default, unless it's set explicitly in config files
+    ///
+    /// Default: true
+    ///
+    /// [deprecated]
```

Comment:
When deprecating something, it is good practice to point users towards the API they should use from now on.